poolmanager/server/main.go

```diff
@@ -116,6 +115,25 @@ func createGrpcCreds(cert string, key string, ca string) (grpc.ServerOption, err
 	return grpc.Creds(creds), nil
 }
 
+func newPluggableControlPlane() controlplane.ControlPlane {
```

Comment:
Do we know for sure that this instantionalizes the object for sure? I think @rdallman also commented on the bottom of the NoopControlPlane with a similar concern.

We could return a factory as an export, and call the function that was exported.

Reply:
yep,fixed